x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/CategorizationIT.java

```diff
@@ -85,14 +104,12 @@ public void setUpData() {
     }
 
     @After
-    public void tearDownData() {
+    public void cleanup() {
         cleanUp();
-        client().admin().indices().prepareDelete(DATA_INDEX).get();
-        refresh("*");
```

Comment:
There's no point deleting the data index or refreshing here, as all indices are cleaned up between tests by the base class